ghga_validator/cli.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Entrypoint of the package"""
+
+import json
+from pathlib import Path
+from typing import Optional
+
+import typer
+from linkml_validator.plugins.jsonschema_validation import JsonSchemaValidationPlugin
+
+from ghga_validator.core.validation import get_target_class, validate
+
+from .plugins.ref_validation import RefValidationPlugin
```

Comment:
```suggestion
from ghga_validator.plugins import RefValidationPlugin
```
One more thing: Please create an `__init__.py` in the plugins module and import all plugins there.